src/libraries/System.Runtime.Numerics/tests/BigInteger/BigNumberTools.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Numerics;
+using System.Reflection;
+
+namespace BigNumberTools
+{
+    public class Utils
+    {
+        private static TypeInfo InternalNumber
+        {
+            get
+            {
+                if (s_lazyInternalNumber == null)
+                {
+                    Type t = typeof(BigInteger).Assembly.GetType("System.Numerics.BigNumber");
+                    if (t != null)
+                    {
+                        s_lazyInternalNumber = t.GetTypeInfo();
+                    }
+                }
+                return s_lazyInternalNumber;
+            }
+        }
+
+        private static volatile TypeInfo s_lazyInternalNumber;
+
+        public static void RunWithFakeThreshold(string name, int value, Action action)
+        {
+            TypeInfo internalNumber = InternalNumber;
+            if (internalNumber == null)
```

Comment:
There is no official rule for this, but I would actually use the new language feature.
```cs
if (internalNumber is null)
```

Reply:
This is also the same as the comment about nameof above. That is, whether we should fix both of them or combine two classes.